BotwActorTool.GUI/BotwActorTool.GUI.csproj

```diff
@@ -1,14 +1,15 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
Is this a compile-time necessity? (Are we or our dependencies using net7.0-only features?)
Are there runtime implications for this? (Do users need to install anything extra to run the program?)

No need to undo it, now that the work is done, just generally a good idea to have a cost-benefit analysis in place before changing frameworks

Reply:
Some of my other libraries already use `.net7`, and I don't want to lower them.

As for user experience, most users will need to install the `.net7` runtime, but they'd need the `.net6` runtime anyway, so there's little difference; and installing .NET is a trivial task. ~Unlike python installs...~